content/cloud-docs/api-docs/providers.mdx

```diff
@@ -222,6 +241,45 @@ curl \
   https://app.terraform.io/api/v2/organizations/my-organization/registry-providers
 ```
 
+### Sample Response (private provider)
+
+```json
+{
+  "data": {
+    "id": "prov-cmEmLstBfjNNA9F3",
+    "type": "registry-providers",
+    "attributes": {
+      "name": "aws",
+      "namespace": "hashicorp",
+      "registry-name": "private",
+      "version-statuses": [],
+      "created-at": "2022-02-11T19:16:59.533Z",
+      "updated-at": "2022-02-11T19:16:59.533Z",
+      "permissions": {
+        "can-delete": true
+      }
+    },
+    "relationships": {
+      "organization": {
+        "data": {
+          "id": "my-organization",
+          "type": "organizations"
+        }
+      },
+      "versions": {
+        "data": [],
+        "links": {
+          "related": "/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws"
```

Comment:
Same namespace inconsistency here -- perhaps change hashicorp to my-organization to match the rest of the response/request.

Reply:
thanks for catching these! I will change to 'hashicorp' as it will make it more consistent with the previous documentation for public providers